tests/test_utility/test_abc.py

```diff
@@ -0,0 +1,45 @@
+from unittest import mock
+
+import pytest
+
+from PyQt5.QtCore import QObject, pyqtSignal
+from volumina.utility import abc
+
+
+class TestQAbc:
+    class IObject(abc.QABC):
+        @abc.abstractmethod
+        def foo(self):
+            ...
+
+    class ISignal(abc.QABC):
+        signal = abc.abstractsignal()
+
+    def test_type_error_on_undefined_abstract_method(self):
+
+        with pytest.raises(TypeError):
+            self.IObject()
+
+    def test_abstract_signal_raises_type_error(self):
+        with pytest.raises(TypeError):
+            self.ISignal()
+
+    def test_abstract_signal_raises_type_error(self):
```

Comment:
is this some pytest dark magic that makes multiple methods with the same name work? In any case, I'd prefer not to have that.